lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
```

Comment:
Its the other way around, the ```module ClassMethods``` methods are the dsl level methods collecting the data, ```base.class_attribute :blah``` is carrying the values forward into the parent/mixin class